src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala

```diff
@@ -241,24 +241,28 @@ class PostgresDirectoryDAO(protected val dbRef: DbReference,
     groupId match {
       case group: WorkbenchGroupName => groupPKQueryForGroup(group)
       case policy: FullyQualifiedPolicyId => groupPKQueryForPolicy(policy)
-      case _ => throw new WorkbenchException(s"unexpected WorkbenchGroupIdentity $groupId")
```

Comment:
Why remove this exception?

Reply:
because the cases are already exhaustive